<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalData</class>
 <widget class="QWidget" name="PersonalData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PersonalData</string>
  </property>
  <widget class="QWidget" name="widget_radar" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>371</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_detailedData" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>401</width>
     <height>241</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_detailedData">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>1</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_gameStyle" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_gameStyle">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>1</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
